### ISSUE TITLE
svg切换，fill属性填了url值会覆盖到切换之后的svg上面

### ISSUE THREAD

**Issue by xiaolieask** [react-native-oh-library/react-native-harmony-svg#212]
用例代码如下：
```
import * as React from "react";
import {
    View,
    Text,
    StyleSheet,
    TextInput,
    TouchableOpacity,
    Linking,
    Image
} from "react-native";
import { SvgXml } from "react-native-svg";


const Svgs = {
    icon_m_选择: `<svg width="240" height="240" viewBox="0 0 240 240" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M208.696 120C208.696 168.986 168.986 208.696 120 208.696C71.0151 208.696 31.3047 168.986 31.3047 120C31.3047 71.0151 71.0151 31.3047 120 31.3047C168.986 31.3047 208.696 71.0151 208.696 120ZM159.284 105.856C162.998 101.45 162.437 94.8666 158.03 91.1523C153.624 87.4381 147.041 87.9992 143.327 92.4056L115.174 125.804L95.7689 107.917C91.5315 104.011 84.9301 104.279 81.0241 108.517C77.1181 112.754 77.3868 119.356 81.6242 123.262L109.05 148.542C111.144 150.473 113.936 151.463 116.778 151.284C119.62 151.105 122.266 149.773 124.101 147.595L159.284 105.856Z" fill="url(#paint0_linear_14651_25106)"/><defs><linearGradient id="paint0_linear_14651_25106" x1="45.284" y1="100.573" x2="120" y2="208.696" gradientUnits="userSpaceOnUse"><stop stop-color="#FD573B"/><stop offset="1" stop-color="#FF3A3A"/></linearGradient></defs></svg>`,
    icon_p_未选择: `<svg width="240" height="240" viewBox="0 0 240 240" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M208.696 120C208.696 168.986 168.986 208.696 120 208.696C71.0151 208.696 31.3047 168.986 31.3047 120C31.3047 71.0151 71.0151 31.3047 120 31.3047C168.986 31.3047 208.696 71.0151 208.696 120ZM159.284 105.856C162.998 101.45 162.437 94.8666 158.03 91.1523C153.624 87.4381 147.041 87.9992 143.327 92.4056L115.174 125.804L95.7689 107.917C91.5315 104.011 84.9301 104.279 81.0241 108.517C77.1181 112.754 77.3868 119.356 81.6242 123.262L109.05 148.542C111.144 150.473 113.936 151.463 116.778 151.284C119.62 151.105 122.266 149.773 124.101 147.595L159.284 105.856Z"/></svg>`,
}
export default OpenLinkView = () => {
    const [isSelect, setIsSelect] = React.useState<boolean>(false);

    const onPress = React.useCallback(() => {
        setIsSelect(!isSelect);
    }, [isSelect]);

    return (
        <View style={styles.bg}>
            <TouchableOpacity style={styles.btn} onPress={onPress}>
                <Text>点击切换图标</Text>
            </TouchableOpacity>

            {isSelect ? (
                <SvgXml
                    xml={Svgs.icon_m_选择}
                    width={30}
                    height={30}
                    style={{
                        position: "absolute",
                        right: 8,
                        top: 8
                    }}
                />
            ) : (
                <SvgXml
                    xml={Svgs.icon_p_未选择}
                    fill={"#DCDDDE"}
                    width={40}
                    height={40}
                    style={{
                        position: "absolute",
                        right: 8,
                        top: 8,
                        backgroundColor: "yellow"
                    }}
                />
            )}
        </View>
    );
};

const styles = StyleSheet.create({
    bg: {
        justifyContent: "center",
        alignItems: "center",
        backgroundColor: "#ffffff"
    },
    input: {
        backgroundColor: "gray",
        height: 30,
        width: 200,
        margin: 20
    },
    btn: {
        backgroundColor: "red",
        padding: 10,
        margin: 30
    }
});
```
开始显示：
![image](https://github.com/react-native-oh-library/react-native-harmony-svg/assets/26913643/6b250bc5-0d91-48de-ad1e-6908ff2bea99)
切换之后：
![image](https://github.com/react-native-oh-library/react-native-harmony-svg/assets/26913643/b32d4e16-b13c-4a30-b578-6603c799ab89)
再切换回来：
![image](https://github.com/react-native-oh-library/react-native-harmony-svg/assets/26913643/0bfbd8da-2b62-4398-8f58-f61999e4dbc7)

**Comment by Louis-C7** [comment]
fix by [PR#208](https://github.com/react-native-oh-library/react-native-harmony-svg/pull/208)

**Comment by Louis-C7** [comment]
fix in PR#217
